### ISSUE TITLE
Scrollbar support for CupertinoScrollbar.controller

### ISSUE THREAD

**Issue by maxlapides** [flutter/flutter#45025]
I want to take advantage of `CupertinoScrollbar` scrollbar dragging, introduced in iOS 13. Unfortunately, the adaptive `Scrollbar` widget doesn't yet support any way to do this. So, I am current doing this as a workaround:

```dart
Platform.isIOS
  ? CupertinoScrollbar(
      controller: controller,
      child: customScrollView,
    )
  : Scrollbar(
      child: customScrollView,
    );
```

**Comment by maxlapides** [comment]
I also opened #45026 as a separate ticket because these are separate feature requests, but they are related in that they are both scrollbar-related feature requests.

**Comment by KagurazakaHanabi** [comment]
It's fixed in 0120c414fbc5d11c503e2091f420c02ffd6dda67
What's your flutter version?

**Comment by maxlapides** [comment]
Oh nice! I'm on the beta channel, currently `v1.10.7` so it doesn't have this update. But I can see it was landed and that this ticket is a duplicate of #37893 so I'll go ahead and close this ticket. Thanks!
